features/step_definitions/user_steps.rb

```diff
@@ -168,10 +168,16 @@
   ActionMailer::Base.deliveries.length.should == n.to_i
 end
 
+Then /^I should see "([^\"]*)" in the last sent email$/ do |text|
```

Comment:
While this check doesn't hurt, I think it's better to use a regex for `( not)?` and combine it with the existing check below, see [here](https://github.com/diaspora/diaspora/blob/33ad411bbd0d48652d161a60627657f75d467292/features/step_definitions/notifications_steps.rb#L9-L18) for an example with a regex for "not".